settings.py

```diff
@@ -94,11 +94,22 @@
         'content_host': 'https://stable.dev.lcip.org',
         'oauth_host': 'https://oauth-stable.dev.lcip.org/v1',
         'profile_host': 'https://stable.dev.lcip.org/profile/v1',
-        'redirect_url': 'http://localhost:3000/api/v3/accounts/authenticate/',
+        'redirect_url': 'http://olympia.test/api/v3/accounts/authenticate/?config=local', # noqa
```

Comment:
similarly here - you can't update it.  Also, localhost is correct - it's for when addons-frontend is run separately from addons-server and running on localhost.

Reply:
ooops, I think I messed up something because I did not intend to change this value that way :/